support/util/networking.go

```diff
@@ -70,6 +70,15 @@ func KASPodPortFromHostedCluster(hc *hyperv1.HostedCluster) int32 {
 	return 6443
 }
 
+// APIPortForLocalZone returns the port used by processes within a private hosted cluster
+// to communicate with the KAS via the api.<hc-name>.hypershift.local host.
+func APIPortForLocalZone(isLBKAS bool) int32 {
+	if isLBKAS {
+		return 6443
```

Comment:
let's follow up to pick this numbers from consts file in addition to the ones we already define